container-engine-lib/lib/backend_impls/kubernetes/kubernetes_manager/kubernetes_manager.go

```diff
@@ -1148,10 +1154,6 @@ func (manager *KubernetesManager) GetPod(ctx context.Context, namespace string,
 	if err != nil {
 		return nil, stacktrace.Propagate(err, "Failed to get pod with name '%s'", name)
 	}
-	deletionTimestamp := pod.GetObjectMeta().GetDeletionTimestamp()
```

Comment:
We need to be able to retrieve information about a pod marked for deletion so we can wait for its termination.